### DISCUSSION TITLE
CORS problem with GlitchTip

### DISCUSSION THREAD

**Discussion by abogapp-dev** [caprover/caprover#2105] [General]
I am trying to connect to the GlitchTip service hosted in a Digital Ocean VPC, but when I try to configure it from a local nextjs project, I am getting a CORS error as a response, I tried to modify the ngnix file of the GlitchTip instance, but still the communication was not achieved, any idea how to solve, the GlitchTip version, is the one that appears as latest to date.

Thank you all for taking the time to read the post.

**Comment by EmilMoe** [comment]
Did you solve it?

**Comment by githubsaturn** [comment]
I recommend reaching out to GlitchTip developers. Perhaps there is a settings somewhere where you need to whitelist your own domains.
